### ISSUE TITLE
BetterPlayerCupertinoControls throws error on VideoPlayerValue

### ISSUE THREAD

**Issue by charlesluzar** [jhomlala/betterplayer#128]
**Describe the bug**
Implemented BetterPlayer following the normal video player convention. Works perfectly on Android. However, tests on an iOS device throw an error that _latestValue is null.

**To Reproduce**
```
class ContentVideoBPCard extends StatefulWidget implements ContentCard {
  final ContentVideo _content;

  const ContentVideoBPCard({@required Content content}) : _content = content;

  @override
  _ContentVideoBPCardState createState() => _ContentVideoBPCardState();
}

class _ContentVideoBPCardState extends State<ContentVideoBPCard> {
  BetterPlayerController _controller;

  @override
  void initState() {
    BetterPlayerConfiguration _config = BetterPlayerConfiguration(
      aspectRatio: 16 / 9,
      fit: BoxFit.contain,
      controlsConfiguration: BetterPlayerControlsConfiguration(
        enableOverflowMenu: false,
        enableSkips: false,
      ),
    );
    BetterPlayerDataSource _dataSource = BetterPlayerDataSource(
      BetterPlayerDataSourceType.NETWORK,
      this.widget._content.url.toString(),
    );
    _controller = BetterPlayerController(_config);
    _controller.setupDataSource(_dataSource);
    super.initState();
  }

  @override
  Widget build(BuildContext context) {
    return Column(
      crossAxisAlignment: CrossAxisAlignment.start,
      children: [
        Padding(
          padding: const EdgeInsets.only(bottom: 18),
          child: Text(
            widget._content.title,
            style: Theme.of(context).textTheme.contentTitle,
          ),
        ),
        AspectRatio(
          aspectRatio: 16 / 9,
          child: BetterPlayer(controller: _controller),
        ),
      ],
    );
  }
```
* On Android, all functionality works as expected.
* When we load a screen with the widget above on an iOS device and play the video it works.
* When we load a screen with the widget above on an iOS device, play the video and then click the full screen button it throws an error that _latestValue is null briefly before that error rectifies and it continues playing.
* When we load a screen with the widget above on an iOS device and immediately click the full screen button it throws an error that _latestValue is null. The user cannot recover from that error.

**Expected behavior**
An error should not be thrown on iOS devices.

**Output**
```
The following assertion was thrown building BetterPlayerCupertinoControls(dirty, dependencies: [BetterPlayerControllerProvider, MediaQuery], state: _BetterPlayerCupertinoControlsState#34e59):
Latest value can't be null
'package:better_player/src/controls/better_player_controls_state.dart':
Failed assertion: line 185 pos 12: 'latestValue != null'

The relevant error-causing widget was
MaterialApp
lib/app/app.dart:30
When the exception was thrown, this was the stack
#2      BetterPlayerControlsState.isLoading
package:better_player/…/controls/better_player_controls_state.dart:185
#3      _BetterPlayerCupertinoControlsState.build
package:better_player/…/controls/better_player_cupertino_controls.dart:69
#4      StatefulElement.build
package:flutter/…/widgets/framework.dart:4792
#5      ComponentElement.performRebuild
package:flutter/…/widgets/framework.dart:4675
#6      StatefulElement.performRebuild
package:flutter/…/widgets/framework.dart:4847
...

```

**Flutter doctor**
```
[✓] Flutter (Channel stable, 1.22.3, on Mac OS X 10.15.7 19H15, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
[✓] Android Studio (version 4.0)
[✓] VS Code (version 1.51.0)
[✓] Connected device (1 available)
```

Note: 

**Better Player version**
- Version: 0.0.33

**Smartphone (please complete the following information):**
 - Device: iPhone 11 Simulator (We confirmed this same behavior on a physical iPhone 11)
 - OS: iOS 14.1

**Additional context**
The relevant method invocation causing the error is the check for `_latestValue.hasError` from `better_player_cupertino_controls.dart` below...

```
  @override
  Widget build(BuildContext context) {
    _betterPlayerController = BetterPlayerController.of(context);

    if (_latestValue.hasError) {
      return _buildErrorWidget();
    }

    final backgroundColor = _controlsConfiguration.controlBarColor;
    final iconColor = _controlsConfiguration.iconsColor;
    _betterPlayerController = BetterPlayerController.of(context);
    _controller = _betterPlayerController.videoPlayerController;
    final orientation = MediaQuery.of(context).orientation;
    final barHeight = orientation == Orientation.portrait
        ? _controlsConfiguration.controlBarHeight
        : _controlsConfiguration.controlBarHeight + 17;
    final buttonPadding = orientation == Orientation.portrait ? 16.0 : 24.0;
    _wasLoading = isLoading(_latestValue);
```

**Comment by jhomlala** [comment]
Fixed in Better Player 0.0.34.

**Comment by charlesluzar** [comment]
@jhomlala thank you for the prompt fix!
